xml/html_title.xml

```diff
@@ -2876,7 +2874,6 @@
     <example>Rundeck - Login</example>
     <param pos="0" name="service.vendor" value="Rundeck"/>
     <param pos="0" name="service.product" value="Rundeck"/>
-    <param pos="0" name="service.cpe23" value="cpe:/a:rundeck:rundeck:-"/>
```

Comment:
It looks like NIST deprecated `a:rundeck:rundeck` in favor of `a:pagerduty:rundeck` on September 1, 2021. We should probably add a remap if we don't want to change the vendor.  This FP and another were added by HD 19 months ago via PR #269